pkgs/tools/networking/openssh/common.nix

```diff
@@ -8,6 +8,9 @@
 }:
 
 { lib, stdenv
+# This *is* correct, though unusual. as a way of getting krb5-config from the
+# pacakge without splicing See: https://github.com/NixOS/nixpkgs/pull/107606
```

Comment:
```suggestion
# package without splicing See: https://github.com/NixOS/nixpkgs/pull/107606
```

Good idea to add a comment. Could we use target or buildPackages to make it even clearer?

Reply:
Sure - I replicated what was there before since that was "known good" and the original PR mentioned it was a correct use-case, but I'll give it a shot with buildPackages

Reply:
Running with "buildPackages" instead of "pkgs" fails with the same error as the unpatched version. I'll admit that I'm new to the whole cross compiling on nix business, so I don't quite understand why.

Reply:
mmm. I think this fix is fine for now and we maybe investigate that later.